uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Upgrader.php

```diff
@@ -920,6 +920,39 @@ public function upgrade_1037() {
     return TRUE;
   }
 
+  /**
+   * Set default number of days before document expiry when the clone document
+   * should be created = 90 days If this field is blank, update it to 90 days.
+   *
+   * @return bool
+   */
+  public function upgrade_1038() {
+    $optionValue = civicrm_api3('OptionValue', 'get', [
+      'sequential' => 1,
+      'option_group_id' => 'ta_settings',
+      'name' => "days_to_create_a_document_clone",
+    ]);
+    if (empty($optionValue['id'])) {
+      civicrm_api3('OptionValue', 'create', [
+        'option_group_id' => 'ta_settings',
+        'name' => 'days_to_create_a_document_clone',
+        'label' => is_callable('t') ? t('Renewed document creation date offset (days)') : 'Renewed document creation date offset (days)',
```

Comment:
Any reason for not using the translation function without checking if its callable?
I think there's no need for this, just use it. Its a civi function and should always be callable.

Reply:
I copied from another upgrader and wondered if i should change it... But will change now